### ISSUE TITLE
Handling of optional b64 encoded data incorrect

### ISSUE THREAD

**Issue by jwag956** [duo-labs/py_webauthn#168]
Using pydantic 2.0 - the code in structs::_pydantic_v2_validate_bytes_fields()
has the line:

```
            if field.annotation != bytes:
                return v
```

However for user_handle - the field annotation is: typing.Optional[bytes]
so it simply returns without decoding....

**Comment by MasterKale** [comment]
Hello @jwag956, thanks for reporting this. I won't have a whole lot of time to work on this till after mid-September so it'll be a bit before I can get a fix in for this. Contributions are welcome, if anyone comes by and spots this.

**Comment by jwag956** [comment]
Did a first cut and testing/fixing this - alas I think trying to check types is going to be error-prone:
https://stackoverflow.com/questions/56832881/check-if-a-field-is-typing-optional

I believe that in earlier versions, py_webauthn had a list of fields that MUST be base64 encoded and based decoding on the field name rather than type?

**Comment by MasterKale** [comment]
Hello @jwag956, if I move ahead with #195 I'm going to end up closing this issue out as Pydantic won't be in the mix anymore. I'll keep you posted.

**Comment by MasterKale** [comment]
I've merged #195 which removed Pydantic as a dependency of this project so I'm closing out any PR's and issues related to Pydantic as they're no longer needed.
